js/foundation.magellan.js

```diff
@@ -109,18 +110,13 @@ class Magellan {
    * @function
    */
   scrollToLoc(loc) {
-    // Do nothing if target does not exist to prevent errors
-    if (!$(loc).length) {return false;}
     this._inTransition = true;
-    var _this = this,
-        scrollPos = Math.round($(loc).offset().top - this.options.threshold / 2 - this.options.barOffset);
-
-    $('html, body').stop(true).animate(
-      { scrollTop: scrollPos },
-      this.options.animationDuration,
-      this.options.animationEasing,
-      function() {_this._inTransition = false; _this._updateActive()}
-    );
+    var _this = this;
+
+    Foundation.SmoothScroll.scrollToLoc(loc, this.options, function() {
```

Comment:
I know, this is why I am talking about `SmoothScroll.defaults` and not `this.options`. `SmoothScroll.defaults` are the default options of the Class, not of the instantiated object.